Serialized request execution sample (#18)
scalar-labs/scalardl-node-client-sdk: example/binary_request_execution/README.md

> # Binary request execution sample

Reviewer: Can you change the directory name to `serialized_request` ?

This sample code is to demonstrate that the node sdk can be used to execute Scalar DL binary request build by another application using any Scalar sdk.

![](img/samplecode.png)

In conclusion, the listContracts request sent by Web SDK can be executed by ScalarDLT invoked from node sdk.

## Project Prerequisite
Make sure your ScalarDLT server is up and running by referring to [Scalar DL official documentation](https://scalardl.readthedocs.io/en/latest/). 

> `ClientProperties` and `WebService` can be initiated as follows:

Reviewer 2: ```suggestion
`Web-client` and `Web-service` Scalar DL sdk are initialized as follows :
```

```

> // web-client

Reviewer 2: After all is done, please update the clientProperties below to reflect the changes on the code

      'MIICizCCAjKgAwIBAgIUMEUDTdWsQpftFkqs6bCd6U++4nEwCgYIKoZIzj0EAwIw\n' +
      'bzELMAkGA1UEBhMCSlAxDjAMBgNVBAgTBVRva3lvMQ4wDAYDVQQHEwVUb2t5bzEf\n' +
      'MB0GA1UEChMWU2FtcGxlIEludGVybWVkaWF0ZSBDQTEfMB0GA1UEAxMWU2FtcGxl\n' +
      'IEludGVybWVkaWF0ZSBDQTAeFw0xODA5MTAwODA3MDBaFw0yMTA5MDkwODA3MDBa\n' +
      'MEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIEwpTb21lLVN0YXRlMSEwHwYDVQQKExhJ\n' +
      'bnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNC\n' +
      'AAQEa6Gq6bKHsFU2pw0oBBCKkMaihSlRG97Z07rqlAKCO1J+7uUlXbRdhZ2uCjRj\n' +
      'd5cSG8rSWxRE703Ses+JBZPgo4HVMIHSMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUE\n' +
      'DDAKBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0GA1UdDgQWBBRDd2MS9Ndo68PJ\n' +
      'y9K/RNY6syZW0zAfBgNVHSMEGDAWgBR+Y+v8yByDNp39G7trYrTfZ0UjJzAxBggr\n' +
      'BgEFBQcBAQQlMCMwIQYIKwYBBQUHMAGGFWh0dHA6Ly9sb2NhbGhvc3Q6ODg4OTAq\n' +
      'BgNVHR8EIzAhMB+gHaAbhhlodHRwOi8vbG9jYWxob3N0Ojg4ODgvY3JsMAoGCCqG\n' +
      'SM49BAMCA0cAMEQCIC/Bo4oNU6yHFLJeme5ApxoNdyu3rWyiqWPxJmJAr9L0AiBl\n' +
      'Gc/v+yh4dHIDhCrimajTQAYOG9n0kajULI70Gg7TNw==\n-----END CERTIFICATE-----\n',
};
const webService = new ClientServiceWithBinary(properties);
```

> ## Spin up web-service

Reviewer: ```suggestion
## Run the sample
```

Author: But this part is more toward running up the backend server of the sample code, Should it be `Run the sample code backend server`?

```

> cd web-service;

Reviewer: Please add a simple explanation for each line.

> ## Spin up web-client

Reviewer 2: ```suggestion
## Spin up web-service
```

```

> node node-server.js

Reviewer 2: ```suggestion
cd web-service;
npm install;
node node-server.js
```

```
### Send Scalar DL requests from the browser

> The Scalardl web client request can be invoked via the html file [here](./web-client/index.html)

Reviewer 2: ```suggestion
The Scalar DL web client request can be invoked via the html file [here](./web-client/index.html)
```
